logging/logrus/server_interceptors_test.go

```diff
@@ -50,19 +50,22 @@ func (s *logrusServerSuite) TestPing_WithCustomTags() {
 	assert.Len(s.T(), msgs, 2, "two log statements should be logged")
```

Comment:
I think we should change check here and 2 lines above (`NoError`) to `require` otherway it can cause runtime errors in the code below.